build.gradle

```diff
@@ -8,12 +11,13 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.2'
+        
+        classpath 'com.android.tools.build:gradle:7.2.2'
```

Comment:
Did you have to upgrade the Gradle version in order to make the demo work? I was planning to do this anyway for v5, but we should probably do it in a separate PR.